src/components/common/DropdownOrder/DropdownOrder.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import arrowUpImage from '../../../images/icons/arrowUp.svg';
+import arrowDownImage from '../../../images/icons/arrowDown.svg';
+import styles from './DropdownOrder.module.css';
+
+export default function DropdownOrder({ children, List, data }) {
```

Comment:
```suggestion
export default function DropdownOrder({ children, list, data }) {
```
보통 대문자로 시작하는 것은 컴포넌트의 이름이기 때문에, 이 부분은 변경해주세요!

Reply:
네! 수정하였습니다!